rcl/test/rcl/test_rmw_impl_id_check.py.in

```diff
@@ -5,7 +5,7 @@ import os
 from launch import LaunchDescription
 from launch import LaunchService
 from launch.actions import ExecuteProcess
-from launch_testing import LaunchTestService
+from launch_testing.legacy import LaunchTestService
```

Comment:
Is this any particular reason why this test wasn't migrated?
Only for curiosity, I think it's ok migrating it later.

Reply:
Oh my, because I missed it :o

Good catch!